pkgs/by-name/se/sequoia-sq/package.nix

```
@@ -10,22 +10,23 @@
   capnproto,
   installShellFiles,
   openssl,
+  cacert,
   sqlite,
 }:
 
 rustPlatform.buildRustPackage rec {
```

migrating this package to use finalAttrs would be good
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

---

I don't have confidence in when the best timing of these migrations is. Personally, I want to focus on fixing the build error in this PR.

---

Thanks for this suggestion, I put the change in revised commit https://github.com/NixOS/nixpkgs/pull/392424/commits/097ad426efb6a3f6cd197fc9d5e03239cf6741b8 :pray: